### DISCUSSION TITLE
Hyprdots not working with nvidia 470 driver

### DISCUSSION THREAD

**Discussion by Coder-Vasen** [prasanthrangan/hyprdots#1897] [Q&A]
I am well aware that there is no support for wayland for the specified nvidia driver. Is there is anyway in hyprdot configuration to use **X11 instead of wayland.** 

The problem I am facing is looping into the login screen. When I remove the `nvidia_drm.modeset=1`  command it is logging in. Then I understood the nvidia driver is the cause of the

**Comment by kRHYME7** [comment]
No, we are using hyprland as the compositor/window manager and it only supports Wayland. You can try to compile the legacy renderer version of hyprland though. But without blurs etc. you can also try to just use the nouveau driver and see if Hyprland loads in there.
